src/core_plugins/timelion/public/directives/expression_directive.js

```diff
@@ -47,8 +47,9 @@ app.directive('timelionExpressionInput', function ($compile, $http, $timeout) {
       sheet: '=',
     },
     replace: true,
+    transclude: true,
```

Comment:
This is fixing the original bug. It allows us to inject any kind of input element (e.g. `<input>` or `<textarea`) into this directive, and provide suggestions based on the user's input. In my multiline expression PR, I'll inject a `<div contenteditable="true">` into it.